.github/pull_request_template.md

```diff
@@ -1,5 +1,24 @@
 ## Infos
 
+<!--
+
+### Pull Request Title Convention
+
```

Comment:
pq tá comentado?

Reply:
It is commented to appear as an alert in the template during editing but not to be displayed in the final PR.